test/index.coffee

```diff
@@ -24,3 +26,27 @@ Amen.describe "Haiku9 static-site generation", (context) ->
 
     context.test "Compile CoffeeScript files", ->
       assert yield exists "build/site.js"
+
+  context.test "Run the serve task", ->
+    context.test "Starts a local server on the default port", ->
+      command = join __dirname, "..", "bin", "h9"
+      child = childProcess.spawn(command, ["serve"])
+
+      # Give the server up to 5 seconds to spin up (but resolve immediately when you can)
+      start = now = new Date()
```

Comment:
No, not dumb at all! No magic, this just initializes both `start` and `now` to the current time, and then `now` gets bumped up to the new current time at the end of each loop, until it gets to be 5 seconds from `start`, and then it fails if we haven't gotten a response by then.